vuepress/docs/v6.3.2/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -1,344 +1,233 @@
-      }
-    }
-
-# Technical Documentation
-
-## Micro Frontend Overview
-
-### Task forms
-
-Widgets containing form (e.g., task completion form widget) use JSON
-schema to dynamically create forms. Widget implementation uses
-react-jsonshema-form library that converts forms’ JSON schemas into
-React components. Our initial implementation uses Material UI components
-via Material UI theme library rjsf-material-ui as baseline and has
-Entando specific templates, widgets and fields (which are
-react-jsonshema-form terms for parts of forms). In this section you will
-learn about basic setup of the form, but if you would like to learn
-more, please refer to react-jsonshema-form documentation. Themed JSON
-form is created using the withTheme() method from the
-react-jsonschema-form package
-
+ 
+#### Classes
+ 
+`Engine`: represents a BPM engine and exposes the services that are available for that specific implementation. It is intended to be inherited, and the subclass should provide the implementation for each service by calling the superclass constructor with the service implementations as arguments. If any service is not supported, a null value should be passed to the corresponding constructor argument. The engine can provide implementations for service interfaces.
+ 
+#### Interfaces
+ 
+`TaskService`: defines service methods for task retrieval from the BPM engine.
+ 
+`TaskDefinitionService`: defines service methods related to task definition. A task definition specifies which fields or columns are available for all task instances.
+ 
+`TaskCommentService`: defines service methods related to task comment manipulation. It should be implemented if the task comment is supported by the engine.
+ 
+`TaskAttachmentService`: defines service methods to operate on task attachments. It should be implemented if the engine supports file attachment on the task.
+ 
+`TaskFormService`: defines service methods for task form operations, like retrieving the form definition and submitting a form. The Form object can be used to render a form dynamically.
+ 
+`TaskLifecycleService`: defines service methods related to the task lifecycle. The lifecycle operations move the task from one state to another.
+ 
+`TaskLifecycleBulkService`: defines methods for bulk lifecycle operations. Like the TaskLifecycleService, methods here move the task from one state to another, but this interface works with multiple tasks at a time.
+ 
+`ProcessService`: defines service methods for process definitions operations.
+ 
+`ProcessFormService`: defines service methods for process form operations, like retrieving the form definition and submitting a form. The Form object can be used to render a form dynamically.
+ 
+`GroupService`: defines service methods related to groups in the BPM engine.
+ 
+## Technical Guide
+ 
+This page explores PDA plugin structure and functionality in greater detail. The sample code and linked resources instruct the user in the versatility and usability of: 
```

Comment:
The Technical Guide might be better broken out into a child page, and not included in the left nav. I think it's possibly useful but a distraction at this level.